content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -37,13 +37,7 @@ fields is available in the inline
 ## {{% heading "prerequisites" %}}
 
 You need to have a Kubernetes cluster.
-You also need kubectl v1.11 or higher, configured to communicate with your cluster.
 {{< version-check >}}
-Your cluster API server version (eg v1.12) must be no more than one minor
-version away from the version of kubectl that you are using. For example,
-if your cluster is running v1.16 then you can use kubectl v1.15, v1.16
-or v1.17; other combinations
-[aren't supported](/docs/setup/release/version-skew-policy/#kubectl).
```

Comment:
Hey Tim, are these constraints no longer valid? Otherwise the PR looks good to me 🙂

Reply:
We (currently) don't support Kubernetes older than v1.18 - see https://kubernetes.io/docs/home/supported-doc-versions/